src/test/java/core/basesyntax/Utils.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+import core.basesyntax.model.FruitTransaction;
+import java.io.File;
+
+public class Utils {
```

Comment:
does this class follow single responsibility principle?

please make these methods private in corresponding classes